src/load.c

```diff
@@ -112,7 +112,7 @@ static int test_module(struct xmp_test_info *info, HIO_HANDLE *h)
 		if (format_loaders[i]->test(h, buf, 0) == 0) {
 			int is_prowizard = 0;
 
-#if !defined(LIBXMP_CORE_PLAYER) && !defined(LIBXMP_NO_PROWIZARD)
+#if !defined(LIBXMP_NO_PROWIZARD)
```

Comment:
This could be changed back to an `#ifndef` for consistency with the other checks in this file. (I *think* it was `#ifndef` before, anyway...)

Reply:
Done.